conjure-java-core/src/integrationInput/java/com/palantir/product/ListExample.java

```diff
@@ -218,6 +218,13 @@ public Builder primitiveItems(@Nonnull Iterable<Integer> primitiveItems) {
             return this;
         }
 
+        public Builder addAllPrimitiveItems(@Nonnull int... primitiveItems) {
```

Comment:
Technically... if my words in the PR description are to be believed and I am mirroring Immutables, then this method should be generated with a name `primitiveItems`. Very easy change to make if people find that clearer.